.github/workflows/test-publish-dry-run.yaml

```diff
@@ -79,3 +79,15 @@ jobs:
           unzip ./examples/android-gradle-kts/build/outputs/apk/release/android-gradle-kts-release-unsigned.apk -d output
           cat output/assets/sentry-debug-meta.properties
           rm -r output
+
+      - name: Build the release App Bundle and run uploadSentryProguardMappings task
+        run: ./gradlew bundleRelease uploadSentryProguardMappingsRelease
```

Comment:
a note not related to this PR (because the APK test also do it):

our test calls the assemble task and `uploadSentryProguardMappingsRelease`, but the thing is, in the real world people won't call `uploadSentryProguardMappingsRelease` directly so our tests should not rely on it.

if CI passes because `uploadSentryProguardMappingsRelease` was called manually but it does not work if you don't call `uploadSentryProguardMappingsRelease` manually, we have a false positive green CI.

what's about removing these task calls? tests should be green by just calling `assembleRelease` or `bundleRelease`

Reply:
> if CI passes because `uploadSentryProguardMappingsRelease` was called manually but it does not work if you don't call `uploadSentryProguardMappingsRelease` manually, we have a false positive green CI.

Agree.

Ideally I would like to:

```
./gradlew bundleRelease
```

And then verify that the Gradle call invoked `uploadSentryProguardMappingsRelease`.
I haven't found an elegant way to verify that, other than:

```
./gradlew bundleRelease --info | tee output.txt && grep uploadSentryProguardMappingsRelease output.txt
```

That is far from optimal 🤔 

The risk of calling just `bundleRelease` is that you don't verify at all that `uploadSentryProguardMappingsRelease` is invoked. The Gradle execution can succeed and the checks inside the APK will also be green (due to the generate UUID task).

Reply:
mm I agree with the statement, but if the mapping file is inside of the APK or AAB, that means the `uploadSentryProguardMappingsRelease` was invoked and did its job, so I guess I'd be fine with it.

`uploadSentryProguardMappingsRelease` does the task scheduling, plus calling the other tasks and glue them  together, so I'd assume that only calling the `bundle/assebleRelease` plus unzipping the apk/aab and checking the file inside of the assets folder is good enough, wdyt?

Reply:
> that means the uploadSentryProguardMappingsRelease was invoked and did its job

Actually it means that `generateSentryProguardUuidRelease` was run, but not necessarily `uploadSentryProguardMappingsRelease`

Reply:
yeah indeed, `SentryUploadProguardMappingsTask` is only responsible to call sentry-cli with the arguments, let's keep as it is for now then, good point